downloads/swan-lake-release-notes/swan-lake-preview2/RELEASE_NOTE.md

```diff
@@ -243,26 +243,29 @@ The `config:getAsArray` and `config:getAsMap` functions now return immutable val
 
 ### WebSocket
 
-The WebSocket client now supports cookies. A new field `cookies` has been introduced in the `http:WebSocketClientConfiguration` and `http:WebSocketFailoverClientConfiguration` to specify the cookies.
+The WebSocket client now supports cookies. A new field `cookies` has been introduced in the `http:WebSocketClientConfiguration` and `http:WebSocketFailoverClientConfiguration` records to specify the cookies.
 
 ### Runtime API
 
 - The `runtime:timeout` function has been removed. The `runtime:sleep` function can be used as the alternative.
 
-For example, when the following, which previously used the `runtime:timeout` is considered,
+**The previous method:**
+
+Used the `runtime:timeout` as shown below.
 
 ```ballerina
 future<()> f1 = runtime:timeout(50);
 ```
-can be done as follows:
-```ballerina
```

Comment:
This change has removed one example, and now "The new method" points to the next "The `runtime:getProperty` function has been removed. Java interoperability can be used as the alternative to access Java system properties.".

Please fix.

And like I mentioned on the original PR, we can't call this a new method.

Reply:
Fixed.